fdbserver/include/fdbserver/ClusterController.actor.h

```diff
@@ -183,10 +183,15 @@ class ClusterControllerData {
 
 		void setEncryptKeyProxy(const EncryptKeyProxyInterface& interf) {
 			auto newInfo = serverInfo->get();
+			auto newClientInfo = clientInfo->get();
```

Comment:
Am not sure if the change is part of this patch?

Reply:
This is needed to make sure that the client db info has an updated EKP otherwise decryption will fail in dumData. I had this as part of the Snapshot PR but i needed it here as well